demos/2d_compressible_ALA/2d_compressible_ALA.py

```diff
@@ -168,7 +170,7 @@
     energy_conservation_2 = abs(rate_work_against_gravity - rate_viscous_dissipation)
 
     # Calculate L2-norm of change in temperature:
-    maxchange = sqrt(assemble((T - energy_solver.T_old)**2 * dx))
+    maxchange = sqrt(assemble((T - energy_solver.solution_old) ** 2 * dx))
```

Comment:
I think it would be neater if energy solver has an attribute T_old  = solution_old. so the demos/existing user code doesnt have to change.

Reply:
Happy for devs to discuss that too.

Reply:
I think this would be a nice thing to keep for compatibility too. It doesn't cost anything to have the same underlying function with two different names.

Reply:
Added in. Thank you.